eleventy/media.js

```diff
@@ -0,0 +1,43 @@
+import { eleventyImageTransformPlugin } from '@11ty/eleventy-img'
+
+/**
+ * Configures processing of media (images, videos)
+ *
+ * @param {import("@11ty/eleventy/UserConfig")} eleventyConfig
+ */
+export function setupMedia(eleventyConfig) {
+  // Load Eleventy image plugin. In this configuration, it automatically
+  // resizes and compresses the sources of <img>s referenced in output HTML
+  eleventyConfig.addPlugin(eleventyImageTransformPlugin, {
+    // Output image formats
+    formats: ['svg', 'avif', 'webp'],
```

Comment:
IIRC we also had to add `gif` to this list for animated GIFs to work; though WebP also supports animation, so if we're fine with animated WebP and eleventy-img generates them correctly, then we can stick with that.